compiler/src/dmd/declaration.h

```diff
@@ -118,7 +118,6 @@ class Declaration : public Dsymbol
     LINK _linkage;              // may be `LINK::system`; use `resolvedLinkage()` to resolve it
     short inuse;                // used to detect cycles
     uint8_t adFlags;
-    Symbol* isym;               // import version of csym
```

Comment:
you will also need to update `frontend.h`

Reply:
Thanks, done.